### ISSUE TITLE
Context menu does not meet color contrast requirement of 4.5:1

### ISSUE THREAD

**Issue by jayspadie** [Microsoft/monaco-editor#439]
monaco-editor version: 0.8.3
Browser: Any
OS: Windows 10

During Accessibility testing, it was found that the active item in the context menu does not meet the WCAG AA color contrast requirement of 4.5:1, as shown below from the Monaco playground using Chrome.  Here, "Go to Definition" has a contrast ratio of only 2.1:1.  Similarly, the shortcut keys listed for each item also do not meet contrast requirements.

![image](https://cloud.githubusercontent.com/assets/5246869/26078442/91fded82-398d-11e7-8057-dc98c59abaea.png)

**Comment by chrisdias** [comment]
Should be using the High Contrast theme.

**Comment by jayspadie** [comment]
As I understand it, the WCAG 4.5:1 contrast ratio requirement is supposed to be met without having to resort to a high contrast theme.

**Comment by alexandrudima** [comment]
The playground page has a bad selector that selects all `a` tags. Here is how the editor looks like in a page without any CSS interference after the change:

---

### Light theme
![screen shot 2017-06-09 at 16 15 38](https://user-images.githubusercontent.com/5047891/26979387-e3ea9500-4d2e-11e7-9aa7-48844376dc19.png)

#### Labels + Keybindings:
![screen shot 2017-06-09 at 16 16 09](https://user-images.githubusercontent.com/5047891/26979412-f8a9bf52-4d2e-11e7-960e-e4bd0b1fb130.png)

### Focused menu entry:
![screen shot 2017-06-09 at 16 16 38](https://user-images.githubusercontent.com/5047891/26979437-0a2b565a-4d2f-11e7-94a0-4a190bbe4e55.png)


---

### Dark theme
![screen shot 2017-06-09 at 16 12 55](https://user-images.githubusercontent.com/5047891/26979289-95565eec-4d2e-11e7-9da7-01044a7ee07c.png)

#### Labels + Keybindings:
![screen shot 2017-06-09 at 16 13 47](https://user-images.githubusercontent.com/5047891/26979327-ae785e52-4d2e-11e7-8552-1660f607f51c.png)

#### Focused menu entry:
![screen shot 2017-06-09 at 16 15 01](https://user-images.githubusercontent.com/5047891/26979367-d30512c4-4d2e-11e7-820b-d1d9e4426670.png)
